common-test-lib/src/main/java/com/google/gct/idea/testing/TestUtils.java

```diff
@@ -127,6 +129,7 @@ public static MockProject mockProject(@Nullable PicoContainer container) {
     return new MockProject(container, getParentDisposableForCleanup());
   }
 
+  @SuppressFBWarnings(value="DM_STRING_CTOR", justification="Warning due to string concatenation")
```

Comment:
FindBugs warns about calling a String constructor like in a case of String s = new String("str") or SomeFnCall(String("str")). FindBugs suggests to use the string primitive directly. Unfortunately, this warning comes up where strings are concat'ed, e.g., a few lines below, you will see two occurrences of string concats.

It's about negligible performance impact. Perhaps we may blanket disable and forget about this warning forever instead of annotating each method. What's your thought?